webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala

```diff
@@ -144,13 +139,6 @@ class ReportsExecutionService(
     }
   }
 
-  def updateScore(nodeId: NodeId, report: NodeStatusReport): IOResult[Unit] = {
```

Comment:
moved back in `ComputeNodeStatusReportService` which contradicts https://github.com/Normation/rudder/pull/5737 
It is done that way because we really want the queue to manage the I/O load, it is done FOR THAT. That's its point. And now, the services are better slipted appart, so the root cause of the problem in quoted PR should be resolved. 
Still, this will need a lot of attention and test in scaling machine.